src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -2195,7 +2202,7 @@ protected int MarkNamedMethod (TypeDefinition type, string method_name, in Depen
 				if (method.Name != method_name)
 					continue;
 
-				MarkMethod (method, reason, ScopeStack.CurrentScope.Origin);
+				MarkMethod (method, reason, ScopeStack.CurrentScope.Origin, enableReflectionPatternReporting: true);
```

Comment:
There are a lot of places where we pass in `true` - I would probably make that a default parameter value. I know it's a bit error prone (easy to forget about), but I would probably still prefer it over the much longer text everywhere else. Thoughts?